<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
	font: normal 12px Helvetica, Arial, sans-serif;
}
#mainFrame {background: white;}
#upperLabel {
	font: normal 20px Helvetica, Arial, sans-serif;
	color: purple;
}
#lowerLabel {
	color: gray;
}
#loginFrame {
	padding: 9px 10px;
	color: black;
	background: white;
	border: 1px solid #d8d8d8;
	border-radius: 2px;
	padding: 0px 10px;
}
QPushButton {
	padding: 8px 16px;
	font: bold 12px Helvetica, Arial, sans-serif;
	color: #6e6e6e;
	background: #f3f3f3;
	background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #F5F5F5, stop: 1 #F1F1F1);
	border: 1px solid #d8d8d8;
	border-radius: 2px;
}
QPushButton:hover {
	color: #333;
	border-color: #c6c6c6
}
QPushButton:pressed {
	color: #000;
	border: 1px solid #c6c6c6;
	background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #eee, stop: 1 #e0e0e0);
}
QTextEdit {
	padding: 7px 10px;
	color: black;
	background: white;
	border: 1px solid #d8d8d8;
	border-radius: 2px;
}
QListWidget {
	padding: 7px 10px;
	color: black;
	background: white;
	border: 1px solid #d8d8d8;
	border-radius: 2px;
}
QLineEdit {
	padding: 7px 10px;
	color: black;
	background: white;
	border: 1px solid #d8d8d8;
	border-radius: 2px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="chatPage">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="chatHBox">
             <item>
              <layout class="QVBoxLayout" name="chatVBox">
               <item>
                <layout class="QHBoxLayout" name="roomHBox">
                 <item>
                  <widget class="QLineEdit" name="roomLine">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="roomButton">
                   <property name="text">
                    <string>Connect to another room</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTextEdit" name="chatField">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QListWidget" name="userList">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>200</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::NoSelection</enum>
               </property>
               <property name="movement">
                <enum>QListView::Static</enum>
               </property>
               <property name="flow">
                <enum>QListView::TopToBottom</enum>
               </property>
               <property name="layoutMode">
                <enum>QListView::SinglePass</enum>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="viewMode">
                <enum>QListView::ListMode</enum>
               </property>
               <property name="sortingEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="controlsHBox">
             <item>
              <widget class="QLineEdit" name="messageLine">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sendButton">
               <property name="text">
                <string>Send</string>
               </property>
               <property name="default">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sendImageButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Send File</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="infoHBox">
             <item>
              <widget class="QPushButton" name="aboutButton">
               <property name="text">
                <string>About</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="loginPage">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QFrame" name="loginFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="signInLabel">
                <property name="text">
                 <string>Sign in</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="signInHBox">
                <item>
                 <layout class="QVBoxLayout" name="labelVBox">
                  <item>
                   <widget class="QLabel" name="serverLabel">
                    <property name="text">
                     <string>Server:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="nameLabel">
                    <property name="text">
                     <string>Name: </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="editLinesVBox">
                  <item>
                   <widget class="QLineEdit" name="serverNameLine">
                    <property name="minimumSize">
                     <size>
                      <width>170</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>localhost</string>
                    </property>
                    <property name="maxLength">
                     <number>15</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="userNameLine">
                    <property name="maxLength">
                     <number>12</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="loginButtonHBox">
                <item>
                 <spacer name="horizontalSpacerLeft">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="loginButton">
                  <property name="text">
                   <string>Login</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacerRight">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serverNameLine</tabstop>
  <tabstop>userList</tabstop>
  <tabstop>messageLine</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>userNameLine</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>sendImageButton</tabstop>
  <tabstop>aboutButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
